SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -47,6 +47,7 @@ BuildRequires:  kernel-uvm-cvm-devel
 Requires:  kernel-uvm
 Requires:  kernel-uvm-cvm
 Requires:  moby-containerd-cc
+Requires:  %{_libexecdir}/virtiofsd
```

Comment:
More context: 
- https://github.com/microsoft/CBL-Mariner/issues/6656  [Comment](https://github.com/microsoft/CBL-Mariner/issues/6656#issuecomment-1836641985)

Reply:
Suggest to define an explicit package dependency (qemu-virtiofsd)